contracts/Staking.sol

```diff
@@ -145,18 +127,20 @@ contract Staking is IERC20, IERC20Metadata {
         updateRewardPool();
         require(_sender != address(0), "ERC20: transfer from the zero address");
         require(_recipient != address(0), "ERC20: transfer to the zero address");
-
+        
```

Comment:
Should be newline

Reply:
> Should be newline
Can you specify the necessary change?

Reply:
Here we've got unnecessary tab, empty line should be empty

Reply:
@drinkius fixed